vagrant/provisioning/roles/ldap-portal/tasks/main.yml

```diff
@@ -0,0 +1,78 @@
+- name: LDAP required packages
```

Comment:
Somehow it would be better to avoid duplicating code by extracting the common blocks to a task that can be reused by both the platform LDAP and portal LDAP roles.  We can add this ticket to be done later if you can't do it in time for this installation.